pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -1,14 +1,12 @@
-{ stdenv, fetchFromGitHub, curl, dmd, gcc }:
+{ stdenv, fetchurl, curl, dmd, gcc }:
 
 let version = "0.9.24"; in
 stdenv.mkDerivation {
   name = "dub-${version}";
 
-  src = fetchFromGitHub {
-    sha256 = "52ea88d075f3a43c84d8cdc3a8683bcc8133b7db";
-    rev = "v${version}";
-    repo = "dub";
-    owner = "dlang";
+  src = fetchurl {
```

Comment:
How is the the so called sha256 hash above (ie., 52ea88d075f3a43c84d8cdc3a8683bcc8133b7db) generated. 

- Is it the git's commit hash ? (isn't that sha1?)
- Is it the tar.gz archive's sha256 hash.. (that is 64 characters)..

Essentially, I can fix this commit if I knew how to generate the correct hash